android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/ContactsModule.java

```diff
@@ -42,6 +42,25 @@
 public class ContactsModule extends ReactContextBaseJavaModule {
   private static final String TAG = ContactsModule.class.getSimpleName();
 
+  private static final List<String> JUST_ME_PROJECTION = new ArrayList<String>() {{
```

Comment:
maybe something like `BASIC_PROJECTION` would give some more idea of what it is

Reply:
check